exe_4.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+
+
+def zero_and_one_matrix(size:int)->np:
+    """
+
+    :param size: Matrix size.
+    :return: The matrix in which the elements on the borders will be equal to 1, and inside 0.
+    """
+    matrix = np.ones((size, size))
+    matrix[1:-1, 1:-1] = 0

```

Comment:
This works great, you can also use numpy.pad
https://numpy.org/doc/stable/reference/generated/numpy.pad.html